homeassistant/components/dwd_weather_warnings/sensor.py

```diff
@@ -116,8 +117,10 @@ def extra_state_attributes(self):
 
         if self.entity_description.key == CURRENT_WARNING_SENSOR:
             searched_warnings = self.api.current_warnings
+            data[ATTR_TYPE] = "current"
```

Comment:
So, based on the description in this PR, there are a few things.

1. We want to reduce the usage of state attributes.
2. We want to avoid adding static values into the state machine.
3. This is adding static information into the state machine to support the UI (a third-party one even). We shouldn't be solve UI issues in the backend.

The use of the entity registry in the card is the correct path, or, let the user set the entities they need.


Going to close this PR because of the above reasoning.

../Frenck